script_facade/models/r4/medication_request.py

```diff
@@ -16,51 +16,77 @@ def __init__(self):
         self.source_identifier = None
 
     @classmethod
-    def from_xml(cls, med_dispensed, source_identifier):
+    def from_xml(cls, med_dispensed, source_identifier, script_version):
         """Build a MedicationRequest from a MedicationDispensed xml element object
 
         :param med_dispensed: MedicationDispensed xml element object
 
         """
-        med_order = cls()
-        med_order.source_identifier = source_identifier
 
-        # todo: separate finding/extract into separate steps
-        drug_description = med_dispensed.xpath('.//DrugDescription/text()')[0]
+        # wrap version-specific extraction methods
```

Comment:
I'd recommend the use of a factory method to return the correct instance type.  This would be a function outside the scope of the class.